scripts/FigS4A-B_celltype-diagnostic-plots.R

```diff
@@ -0,0 +1,199 @@
+# This script is used to generate example diagnostic plots for CellAssign and SingleR 
+
```

Comment:
Since much of this code is from the QC, let's state that and link - 

https://github.com/AlexsLemonade/scpca-nf/blob/cf1bdae3a711a37d67aa18ca1b8e26314c8e43f7/templates/qc_report/celltypes_supplemental_report.rmd